.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,25 @@
+name: Dashboard Linting and Formatting
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+      - 'releases/**'
+  pull_request:
+    branches:
+      - master
+      - 'releases/**'
+  merge_group:
+
+jobs:
+  dashboard:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install dependencies and check
+      run: |
+        npm --prefix sky/dashboard install
+        npm --prefix sky/dashboard run lint
```

Comment:
Does this fail if the lint changes anything without a `set -e`?

Reply:
The [shell](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) is run with `base -e` on linux runner by default.